[rospy.client][INFO] 2021-11-24 16:45:53,707: init_node, name[/server_endpoint], pid[18608]
[xmlrpc][INFO] 2021-11-24 16:45:53,708: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2021-11-24 16:45:53,708: Started XML-RPC server [http://wng:39803/]
[rospy.impl.masterslave][INFO] 2021-11-24 16:45:53,708: _ready: http://wng:39803/
[rospy.init][INFO] 2021-11-24 16:45:53,708: ROS Slave URI: [http://wng:39803/]
[rospy.registration][INFO] 2021-11-24 16:45:53,708: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2021-11-24 16:45:53,708: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2021-11-24 16:45:53,808: registered with master
[rospy.rosout][INFO] 2021-11-24 16:45:53,809: initializing /rosout core topic
[rospy.rosout][INFO] 2021-11-24 16:45:53,809: connected to core topic /rosout
[rospy.simtime][INFO] 2021-11-24 16:45:53,810: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2021-11-24 16:45:53,814: Starting server on 192.168.1.146:10000
[rospy.internal][INFO] 2021-11-24 16:45:54,029: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2021-11-24 16:46:09,435: Connection from 192.168.1.146
[rosout][INFO] 2021-11-24 16:46:09,440: RegisterPublisher(execution_check, <class 'panda_move.msg._ExecutionCheck.ExecutionCheck'>) OK
[rospy.internal][INFO] 2021-11-24 16:46:09,492: topic[/execution_check] adding connection to [http://wng:39803/], count 0
[rospy.internal][INFO] 2021-11-24 16:46:09,494: topic[/execution_check] adding connection to [/server_endpoint], count 0
[rospy.core][INFO] 2021-11-24 17:19:01,461: signal_shutdown [signal-2]
[rospy.internal][INFO] 2021-11-24 17:19:01,464: topic[/execution_check] removing connection to /server_endpoint
[rospy.internal][INFO] 2021-11-24 17:19:01,465: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2021-11-24 17:19:01,465: topic[/execution_check] removing connection to http://wng:39803/
[rospy.impl.masterslave][INFO] 2021-11-24 17:19:01,465: signal-2
[rospy.internal][WARNING] 2021-11-24 17:19:01,465: Unknown error initiating TCP/IP socket to wng:38221 (http://wng:39803/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 359, in read_ros_handshake_header
    raise ROSHandshakeException("connection from sender terminated before handshake header received. %s bytes were received. Please check sender for additional details."%b.tell())
ROSHandshakeException: connection from sender terminated before handshake header received. 0 bytes were received. Please check sender for additional details.

[rospy.core][INFO] 2021-11-24 17:19:01,597: signal_shutdown [atexit]
